src/catch2/reporters/catch_reporter_helpers.cpp

```diff
@@ -236,36 +236,50 @@ namespace Catch {
     }
 
     namespace {
-        struct SummaryColumn {
-            SummaryColumn( std::string _label, Colour::Code _colour ):
-                label( CATCH_MOVE( _label ) ), colour( _colour ) {}
-            SummaryColumn addRow( std::uint64_t count ) {
-                ReusableStringStream rss;
-                rss << count;
-                std::string row = rss.str();
-                for ( auto& oldRow : rows ) {
-                    while ( oldRow.size() < row.size() )
-                        oldRow = ' ' + oldRow;
-                    while ( oldRow.size() > row.size() )
-                        row = ' ' + row;
+        class SummaryColumn {
+        public:
+            SummaryColumn( std::string suffix, Colour::Code colour ):
+                m_suffix( CATCH_MOVE( suffix ) ), m_colour( colour ) {}
+
+            SummaryColumn&& addRow( std::uint64_t count ) && {
+                std::string row = std::to_string(count);
+                auto const new_width = std::max( m_width, row.size() );
```

Comment:
you want `m_width` to be `size_t` as well.

Reply:
Right, fixed.